<?xml version="1.0" encoding="UTF-8"?><testrun name="AccountResourceTest.testTransfer" project="bank-example" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testcase name="testTransfer" classname="com.aprilsulu.bank.resources.AccountResourceTest" time="86.364">
    <failure>
      <expected>200</expected>
      <actual>400</actual>org.junit.ComparisonFailure: expected:&lt;[2]00&gt; but was:&lt;[4]00&gt;&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at com.aprilsulu.bank.resources.AccountResourceTest.testTransfer(AccountResourceTest.java:108)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at io.dropwizard.testing.junit.ResourceTestRule$1.evaluate(ResourceTestRule.java:198)&#13;
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.mockito.internal.runners.DefaultInternalRunner$1.run(DefaultInternalRunner.java:68)&#13;
	at org.mockito.internal.runners.DefaultInternalRunner.run(DefaultInternalRunner.java:74)&#13;
	at org.mockito.internal.runners.StrictRunner.run(StrictRunner.java:39)&#13;
	at org.mockito.junit.MockitoJUnitRunner.run(MockitoJUnitRunner.java:161)&#13;
	at org.mockito.runners.MockitoJUnitRunner.run(MockitoJUnitRunner.java:53)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
  </testcase>
</testrun>
